webhook_to_fedora_messaging/config/config.toml

```diff
@@ -0,0 +1,60 @@
+[flaskapp]
```

Comment:
How is this file used? Are those the defaults? Is it an example file? Please add a comment at the top to explain.

Reply:
Ahh. This got added to the commit by mistake. I will remove this. The example config is already there in the project root.